bin/liquidator/src/liquidate.rs

```diff
@@ -370,20 +370,16 @@ impl<'a> LiquidateHelper<'a> {
             .filter_map(|(ti, effective)| {
                 // check constraints for liquidatable assets, see also has_possible_spot_liquidations()
                 let tokens = ti.balance_spot.min(effective.spot_and_perp);
-                let is_valid_asset = tokens >= 1;
+                let is_valid_asset = tokens >= 1 && ti.maint_asset_weight.is_positive();
                 let quote_value = tokens * ti.prices.oracle;
-                // prefer to liquidate tokens with asset weight that have >$1 liquidatable
-                let is_preferred =
```

Comment:
we don't care about preferred assets anymore?

Reply:
Indeed, previously the liquidator preferred to liquidate against assets that had asset weight (and at least $1 in deposits). Now that tokens with asset weight are the only ones that can be liquidated, the logic for it is no longer needed.